src/master/constants.hpp

```diff
@@ -48,6 +48,9 @@ constexpr double MIN_CPUS = 0.01;
 // Minimum amount of memory per offer.
 constexpr Bytes MIN_MEM = Megabytes(32);
 
+constexpr Duration DEFAULT_RECOVERY_TIMEOUT = Minutes(10);
+constexpr float DEFAULT_AGENT_RECOVERY_FACTOR = 0.95;
```

Comment:
if we submit this upstream, we'll need to update the default value. I would suggest to set it to 0.8 (current default in mesos)